docs/learn/test-runner/responsive.md

```diff
@@ -60,53 +59,87 @@ describe('isMobile', () => {
-@media screen and (min-width: 1024px) {
-  .card {
-    background: rgb(255, 0, 0);
-  }
-}
+For demonstration purposes we will write this example using an HTML test. Tests written as HTML are loaded by the test runner directly, a great way to set up some static HTML and CSS for our test.
+
+Create a test file called `my-card.test.html` and set up the basic structure:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      .card {
+        background: rgb(0, 255, 0);
+      }
+
+      @media screen and (min-width: 1024px) {
+        .card {
+          background: rgb(255, 0, 0);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="card"></div>
+  </body>
+</html>
 ```
 
-So how can we test that?
-
-First we set the viewport size.
-Then we create a html fixture which means the nodes will be created and added to the dom (and automatically cleaned up after the test).
-
-Finally we check if our css media query from above actually is working.
-
-```js
-it('mobile has a green background', async () => {
-  await setViewport({ width: 360, height: 640 });
-  const el = await fixture(html` <div class="card"></div> `);
-  expect(getComputedStyle(el).backgroundColor).to.equal('rgb(0, 255, 0)');
-});
-
-it('desktop has a red background', async () => {
-  await setViewport({ width: 1200, height: 1000 });
-  const el = await fixture(html` <div class="card"></div> `);
-  expect(getComputedStyle(el).backgroundColor).to.equal('rgb(255, 0, 0)');
-});
+Next, we can write our tests to change the viewport and check if our media queries are working correctly:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      .card {
+        background: rgb(0, 255, 0);
+      }
+
+      @media screen and (min-width: 1024px) {
+        .card {
+          background: rgb(255, 0, 0);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="card"></div>
+
+    <script type="module">
+      import { runTests } from '@web/test-runner-mocha';
+      import { expect } from '@open-wc/testing';
+      import { setViewport } from '@web/test-runner-commands';
+
+      runTests(() => {
+        it('mobile has a green background', async () => {
+          await setViewport({ width: 360, height: 640 });
+          const el = document.querySelector('.card');
+          const color = getComputedStyle(el).backgroundColor;
+          expect(color).to.equal('rgb(0, 255, 0)');
+        });
+
+        it('desktop has a red background', async () => {
+          await setViewport({ width: 1200, height: 1000 });
+          const el = document.querySelector('.card');
+          const color = getComputedStyle(el).backgroundColor;
+          expect(color).to.equal('rgb(255, 0, 0)');
+        });
+      });
+    </script>
+  </body>
+</html>
 ```
 
-_In this tests we combine js and css so we will need to put the css into the test site_
+To run HTML tests, we just need to include it in the files we pass on to the test runner in our `package.json` scripts:
```

Comment:
```suggestion
To run HTML tests, we need to include it in the files we pass on to the test runner in our `package.json` scripts:
```

just, easy, simply, ... are words that should be avoided in learn/docs

it sort of has the indication that people reading this are not smart enough as it's just that simple so why you need to follow this docs here? 😅

Reply:
Makes sense!